src/main/scala/org/broadinstitute/dsde/firecloud/service/SubmissionService.scala

```diff
@@ -46,6 +46,15 @@ trait SubmissionService extends HttpService with PerRequestCreator with FireClou
         externalHttpPerRequest(requestContext, extReq)
       }
     } ~
+    path("workspaces" / Segment / Segment / "submissions" / Segment) {
+      (workspaceNamespace, workspaceName, submissionId) =>
+        delete { requestContext =>
+          val deleteSubmission: HttpRequest = Delete(FireCloudConfig.Rawls.
+            submissionByIdUrl(workspaceNamespace, workspaceName, submissionId))
+          actorRefFactory.actorOf(Props(new HttpClient(requestContext))) !
+            HttpClient.PerformExternalRequest(deleteSubmission)
```

Comment:
with the PerRequest implementation merged in, this block should change to something like:
```
delete { requestContext =>
          val extReq: HttpRequest = Delete(FireCloudConfig.Rawls.
            submissionByIdUrl(workspaceNamespace, workspaceName, submissionId))
          externalHttpPerRequest(requestContext, extReq)
        }
```
instead of creating the actor explicitly.